integration-tests/src/main/java/io/druid/testing/clients/QueryResourceTestClient.java

```diff
@@ -93,6 +100,7 @@ private String getBrokerURL()
       );
     }
     catch (Exception e) {
+      log.error("exception: " + e.getMessage());
```

Comment:
In my experience, an exception usually passes through a big stack of method calls and we put the log inside every method then pretty much same log(and trace) gets printed in the log files many many times and creates unnecessary noise. That is why, I usually do not log it if it is being propagated.
We might have a special case here but not too sure. I would not block the PR on this but this kind of logging should be exception than the norm.